src/map.js

```diff
@@ -207,6 +207,13 @@ var LMap = L.Map.extend({
             this._zoomBoundLayers[L.stamp(layer)] = layer;
         }
 
+        // ensure logo appears even when mapbox layer added after map is initialized
+        var mapboxLogoControl = this._mapboxLogoControl.getContainer();
+        if (!L.DomUtil.hasClass(mapboxLogoControl)) {
```

Comment:
`hasClass` expects 2 arguments here, so currently the `if` logic here will always be `true`

Reply:
Yep, good catch. Fixed in 36eb4d83